fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/indexes/MultidimensionalIndexTestBase.java

```diff
@@ -496,7 +496,7 @@ void basicReadWithNullsTest(final boolean useAsync, @Nonnull final String storag
     }
 
     void indexReadTest(final boolean useAsync, final long seed, final int numRecords, @Nonnull final String storage,
-                   final boolean storeHilbertValues, final boolean useNodeSlotIndex) throws Exception {
+                           final boolean storeHilbertValues, final boolean useNodeSlotIndex) throws Exception {
```

Comment:
minor:

```suggestion
                       final boolean storeHilbertValues, final boolean useNodeSlotIndex) throws Exception {
```

I think. The main goal here would be to align the `final`s from the two lines